ci/travis/determine_tests_to_run.py

```diff
@@ -0,0 +1,112 @@
+# Script used for checking changes for incremental testing cases
```

Comment:
The linting is failing. Please fix the linting errors. If you're unsure how to run the linting tests locally, let me know.

Reply:
Where would the linting test be located?

Reply:
You can run `ray/scripts/format.sh` to autofix your errors (or identify lint issues)